hudi-common/src/main/java/org/apache/hudi/metadata/FileSystemBackedTableMetadata.java

```diff
@@ -81,23 +81,23 @@ public List<String> getAllPartitionPaths() throws IOException {
       return FSUtils.getAllPartitionFoldersThreeLevelsDown(fs, datasetBasePath);
     }
 
-    return getPartitionPathsWithPrefixes(Collections.singletonList(""));
+    return getPartitionPathsInDirs(Collections.singletonList(""));
   }
 
   @Override
-  public List<String> getPartitionPathsWithPrefixes(List<String> prefixes) {
-    return prefixes.stream().flatMap(prefix -> {
+  public List<String> getPartitionPathsInDirs(List<String> relativePaths) {
```

Comment:
While i'm not a huge fan of `getPartitionPathsWithPrefix` name, i find `getPartitionPathsInDirs` more confusing than the previous one (it's not clear on why we're looking up partition-paths in some directories)

I think we should instead clarify the prefix part of this naming to be proper _path_ prefix, like for ex `getPartitionPathWithPathPrefix` and make a clear call out in the java-doc. WDYT?

Reply:
Sg.  We can go with `getPartitionPathWithPathPrefix`